nntrainer/models/model_loader.cpp

```diff
@@ -70,61 +98,6 @@ int ModelLoader::loadModelConfigIni(dictionary *ini, NeuralNetwork &model) {
   model.batch_size =
     iniparser_getint(ini, "Model:Batch_Size", model.batch_size);
 
-  /** Default to adam optimizer */
-  const char *opt_type = iniparser_getstring(ini, "Model:Optimizer", "adam");
```

Comment:
I think we should provide backward compatibility. There are number of people trying our library.

Reply:
Would this be included to tizen 6.5m1? How about we change to tizen docs as well?

Reply:
will add backward compatibility.